feature/detailnews/src/test/java/kr/co/cotton/feature/detailnews/ExampleUnitTest.kt

```diff
@@ -0,0 +1,17 @@
+package kr.co.cotton.feature.detailnews
+
+import org.junit.Test
+
+import org.junit.Assert.*
```

Comment:
🚫 **[ktlint]** <no-wildcard-imports> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wildcard import (cannot be auto-corrected)